# Parameters for your particle filter

# TA GIVEN PARAMS
#  motion_params:
#    vel_std:    0.05
#    delta_std:  0.5
#    x_std:      0.05
#    y_std:      0.05
#    theta_std:  0.05

#  sensor_params:
#    hit_std:    1.0
#    z_hit:      0.5
#    z_short:    0.05
#    z_max:      0.05
#    z_rand:     0.5


#1st TRY PARAMS
# motion_params:
#   vel_std:    0.01
#   delta_std:  0.1
#   x_std:      0.01
#   y_std:      0.01
#   theta_std:  0.01

# sensor_params:
#   hit_std:    0.6
#   z_hit:      0.2
#   z_short:    0.02
#   z_max:      0.02
#   z_rand:     0.2


# 2nd TRY PARAMS
# motion_params:
#   vel_std:    0.01
#   delta_std:  0.1
#   x_std:      0.01
#   y_std:      0.005
#   theta_std:  0.01

# sensor_params:
#   hit_std:    0.6
#   z_hit:      0.6
#   z_short:    0.02
#   z_max:      0.02
#   z_rand:     0.2


# FINAL PARAMS
 motion_params:
   vel_std:    0.01
   delta_std:  0.1
   x_std:      0.01
   y_std:      0.008
   theta_std:  0.01

 sensor_params:
   hit_std:    0.7
   z_hit:      0.6
   z_short:    0.02
   z_max:      0.01
   z_rand:     0.1

